src/status_im/navigation/screens.cljs

```diff
@@ -136,544 +137,922 @@
     [status-im.navigation.options :as options]
     [status-im.navigation.transitions :as transitions]))
 
+(defn chat-screens
+  []
+  [{:name      :start-a-new-chat
+    :metrics   {:flow   :messenger
+                :track? :true
+                :event  {:id :messenger.new-chat}}
+    :options   {:sheet? true}
+    :component new-chat/view}
+
+   {:name      :chat
+    :metrics   {:flow   :messenger
+                :track? :true
+                :event  {:id :messenger.chat}}
+    :options   {:popGesture false
+                :animations transitions/stack-transition-from-bottom}
+    :component chat/chat}
+
+   {:name      :group-create
+    :metrics   {:flow   :messenger
+                :track? :true
+                :event  {:id :messenger.new-group}}
+    :options   {:sheet?           true
+                :skip-background? true}
+    :component group-create/view}
+
+   {:name      :group-details
+    :metrics   {:flow   :messenger
+                :track? :true
+                :event  {:id :messenger.group-details}}
+    :component group-details/group-details}
+
+   {:name      :group-add-manage-members
+    :metrics   {:flow   :messenger
+                :track? :true
+                :event  {:id :messenger.group-manage-members}}
+    :options   {:sheet? true}
+    :component group-details/add-manage-members}])
+
+(defn community-screens
+  []
+  [{:name      :discover-communities
+    :metrics   {:flow   :community
```

Comment:
I'm on the fence about adding `:flow` now because we aren't using this data yet, or maybe I missed in the diff. Given that `event.id` keyword already has a useful qualification, in the analytics tool this may be sufficient for filtering purposes (not sure).

Reply:
Happy to remove these values 👍 

For context: currently `:flow` is not used for anything, so we do not need that property, it's mostly there as a way to categorise the screens without relying too much on the variable names in the file.

Reply:
+1 for removing them, since we are already categorizing them with vars

Reply:
I also feel we should not add it here. Flow can be `inferred` from the data.